fastlane/spec/actions_specs/get_managed_play_store_publishing_rights_spec.rb

```diff
@@ -21,7 +21,7 @@
         it "found file" do
           expect(UI).to receive(:important).with("To not be asked about this value, you can specify it using 'json_key'").once
           expect(UI).to receive(:input).with(anything).and_return(json_key_path)
-          expect(UI).to receive(:important).with("https://play.google.com/apps/publish/delegatePrivateApp?service_account=#{json_key_client_email}&continueUrl=https://fastlane.github.io/managed_google_play-callback/callback.html")
+          expect(UI).to receive(:important).with("https://play.google.com/apps/publish/delegatePrivateApp?service_account=#{json_key_client_email}&continueUrl=https%3A%2F%2Ffastlane.github.io%2Fmanaged_google_play-callback%2Fcallback.html")
```

Comment:
These tests now reflect https://github.com/fastlane/fastlane/pull/16409/files#r418300953 above - query param should be URL encoded.